Source/WebCore/css/parser/CSSPropertyParserHelpers.cpp

```diff
@@ -4366,12 +4366,41 @@ RefPtr<CSSValue> consumeTextAutospace(CSSParserTokenRange& range)
 {
     //  normal | auto | no-autospace | [ ideograph-alpha || ideograph-numeric || punctuation ] || [ insert | replace ]
     // FIXME: add remaining values;
-    if (auto value = consumeIdent<CSSValueAuto, CSSValueNoAutospace>(range)) {
+    if (auto value = consumeIdent<CSSValueAuto, CSSValueNoAutospace, CSSValueNormal>(range)) {
         if (!range.atEnd())
             return nullptr;
         return value;
     }
-    return nullptr;
+
+    CSSValueListBuilder list;
+    bool seenIdeographAlpha = false;
+    bool seenIdeographNumeric = false;
+
+    while (!range.atEnd()) {
+        auto valueID = range.peek().id();
+
+        if ((valueID == CSSValueIdeographAlpha && seenIdeographAlpha) || ( valueID == CSSValueIdeographNumeric && seenIdeographNumeric))
+            return nullptr;
```

Comment:
This shouldn't be needed, `auto ident = consumeIdent<CSSValueIdeographAlpha, CSSValueIdeographNumeric>(range);` below will return null if an unrecognized ident is parsed.

Reply:
This is for checking that `CSSValueIdeographAlpha CSSValueIdeographAlpha` is invalid

Reply:
Ah I misread, thanks.